shap/plots/_bar.py

```diff
@@ -20,25 +20,31 @@
 # TODO: Have the Explanation object track enough data so that we can tell (and so show) how many instances are in each cohort
 def bar(shap_values, max_display=10, order=Explanation.abs, clustering=None, clustering_cutoff=0.5,
         merge_cohorts=False, show_data="auto", show=True):
-    """ Create a bar plot of a set of SHAP values.
+    """Create a bar plot of a set of SHAP values.
 
-    If a single sample is passed then we plot the SHAP values as a bar chart. If an
-    Explanation with many samples is passed then we plot the mean absolute value for
-    each feature column as a bar chart.
+    If a single sample is passed, then we plot the SHAP values as a bar chart. If an
+    :class:`.Explanation` with many samples is passed, then we plot the mean absolute
+    value for each feature column as a bar chart.
 
 
     Parameters
     ----------
     shap_values : shap.Explanation or shap.Cohorts or dictionary of shap.Explanation objects
-        A single row of a SHAP Explanation object (i.e. shap_values[0]) or a multi-row Explanation
-        object that we want to summarize.
+        A single row of a SHAP :class:`.Explanation` object (i.e. ``shap_values[0]``) or
```

Comment:
Would single backticks be better for general python objects like this? 

```rst
# Suggesting:
`shap_values[0]`

# Rather than:
``shap_values[0]``
```

https://stackoverflow.com/a/22257273

Reply:
Hmm if it is just single backticks, it gets rendered just in italics. So it'll look like *shap_values[0]*.

I've always found that difficult to read, tbh.